src/Functional/monoid.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Widmogrod\Functional;
+
+use Widmogrod\FantasyLand\Monoid;
+use Widmogrod\FantasyLand\Semigroup;
+
+const mempty = 'Widmogrod\Functional\mempty';
+
+/**
+ * mempty :: a
+ *
+ * @param Monoid $a
+ * @return Monoid
+ */
+function mempty(Monoid $a)
+{
+    return $a->getEmpty();
+}
+
+const mappend = 'Widmogrod\Functional\mappend';
+
+/**
+ * mappend :: a -> a -> a
+ *
+ * @param Semigroup $a
+ * @param Semigroup $b
+ * @return Semigroup
+ */
+function mappend(Semigroup $a, Semigroup $b)
```

Comment:
I totally get that `mempty` and `mappend` are coming from Haskell, but since Fantasyland chose other terms it is strange to me to use `concat` on the class and `mappend` here.

Should we stick to one term for each action to avoid confusing the users ?